code/src/Components/RadioButton.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+const RadioButton = (props) => {
+    const buttonValue = props.question.options[props.optionNumber].value
+    return (
+        <>
+            <input
+                name="radio" 
+                id="radio-button" 
+                type="radio"
+                checked={props.source === buttonValue}
+                onChange={() => {props.toChange(buttonValue)}}                       
+            ></input>
+            <label htmlFor="radio-button">
+                {props.question.options[props.optionNumber].label}
+            </label>
```

Comment:
From what I can see, all of your radio-buttons have the same id and label. So, when you click on any label, only first button is checked. Radio buttons should have unique ids and labels should correspond to the ids.